### PR TITLE
Message Event Filter Graph-355

### PR DESCRIPTION
**Pull request by lavabyrd**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module github.com/figment-networks/firehose-cosmos
 
 go 1.17
 
+// replace github.com/figment-networks/proto-cosmos => /Users/markpreston/code/figment/graph-cosmos/proto-cosmos
```
Comment (sosedoff) [inline go.mod:5]: Lets drop the commented out replacement

File: tools/message_type_index.go [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+	}
+
+	if err := s.Run(ctx); err != nil {
+		if !errors.Is(err, stream.ErrStopBlockReached) {
+			return err
+		}
+	}
+	zlog.Info("complete")
+	return nil
+}
```
[the remark this answers is not in the document]
Comment (lavabyrd) [reply tools/message_type_index.go:174]: that makes sense. Is that something we want to do here or shall i create another task for it?

[the remark this answers is not in the document]
Comment (pienkowb) [reply tools/message_type_index.go:174]: Let's create a ticket for that. We have to talk with SF about this first.

**Review by sosedoff — APPROVED** [review]
LGTM, could refactor in separate PR

**Pull request merged**